panther_analysis_tool/main.py

```diff
@@ -569,6 +581,10 @@ def parse_filter(filters: List[str]) -> Dict[str, Any]:
     return parsed_filters
 
 
+def set_env(key: str, value: str) -> None:
+    os.environ[key] = value
+
+
```

Comment:
I don't know that we will ever need to set other env variables, but separating this out allowed me to create a unit test to verify AWS_PROFILE gets set appropriately